tests/features/steps/payment_steps.py

```diff
@@ -74,9 +97,28 @@ def step_impl(context, url):
 def step_impl(context, message):
     assert message in context.resp.data
 
+
 @then('I should not see "{message}"')
 def step_impl(context, message):
-	assert message not in context.resp.data
+    assert message not in context.resp.data
+
+
+@then('I should be returned nothing')
+def step_impl(context):
+    assert context.resp.status_code == status.HTTP_204_NO_CONTENT
+    assert context.resp.data == ''
+
+
+@then('I should see a {http_code} response')
+def step_impl(context, http_code):
+    desired_http_code = getattr(status, http_code)
+    assert context.resp.status_code == desired_http_code
+
+
+@then('I should be returned a {http_code} response')
+def step_impl(context, http_code):
+    desired_http_code = getattr(status, http_code)
+    assert context.resp.status_code == desired_http_code
 
```

Comment:
`I should see {http_code}` and `I should be returned {http_code}` do the same check. Should we just use the "should be returned nothing" step again instead of the `should be returned`? i think that language is more user-friendly then setting expectations in AC. makes sense for those just to be assertions behind the scenes